### ISSUE TITLE
ADD/FIX option to change Caja folder title format wasting space (remove ₓ close button and ...)

### ISSUE THREAD

**Issue by allanlaal** [mate-desktop/caja#1384]
#### Expected behaviour

I see atleast the first few letters of tab titles when I have multiple tabs open

#### Actual behaviour

![Screenshot at 2020-05-06 00-34-53](https://user-images.githubusercontent.com/740826/81118793-39423b00-8f32-11ea-91f0-8ced661644a8.png)

instead 5..6 chars are wasted to display essentially nothing of use

#### Steps to reproduce the behaviour

*

#### MATE general version

1.24

#### Package version

caja                                                        1.24.0-1                                        amd64

#### Linux Distribution

Ubuntu 20.04 LTS

#### Link to downstream report of your Distribution

caja specific

**Comment by ericek111** [comment]
Yep. I believe it was much better on GTK2. It could be caused by the theme, partially. But it would be nice to have the option to hide the close buttons. I rarely ever use them (scroll wheel works wonders).
